prsannots/generic.py

```diff
@@ -7,7 +7,8 @@
 from xml.dom import minidom
 from StringIO import StringIO
 import hashlib
-import pyPdf
+import PyPDF2
+from prsannots.misc import u_raw_input, u_print, u_argv
```

Comment:
I don't think this import is needed.  Perhaps it's left over from diagnostics you were doing?